### PR TITLE
Rewrite without sonata

### PR DESCRIPTION
**Pull request by paul999**
Do not merge.

### CODE REVIEW

File: src/phpBB/StatusSiteBundle/Resources/views/Sites/show.html.twig [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                <td>{{ entity.major }}</td>
+            </tr>
+            <tr>
+                <th>Team</th>
+                <td>{{ entity.team }}</td>
+            </tr>
+        </tbody>
+    </table>
+
+        <ul class="record_actions">
```
Comment (michaelcullum) [inline src/phpBB/StatusSiteBundle/Resources/views/Sites/show.html.twig]: tabbing?

File: src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace phpBB\StatusSiteBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class ChecksControllerTest extends WebTestCase
+{
+    /*
```
Comment (michaelcullum) [inline src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php:9]: Why?

File: src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+        // Create a new entry in the database
+        $crawler = $client->request('GET', '/admin/checks/');
+        $this->assertEquals(200, $client->getResponse()->getStatusCode(), "Unexpected HTTP status code for GET /admin/checks/");
+        $crawler = $client->click($crawler->selectLink('Create a new entry')->link());
+
+        // Fill in the form and submit it
+        $form = $crawler->selectButton('Create')->form(array(
+            'phpbb_statussitebundle_checkstype[field_name]'  => 'Test',
+            // ... other fields to fill
```
Comment (michaelcullum) [inline src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php:23]: what other fields?

File: src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php (hunk 2 above)
Comment (paul999) [inline src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php:9]: Generated code, I guess we can ignore for now.

File: src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php (hunk 3 above)
Comment (paul999) [inline src/phpBB/StatusSiteBundle/Tests/Controller/ChecksControllerTest.php:23]: same here, auto generated code.

**Pull request merged**